index.html

```diff
@@ -8,7 +8,6 @@
 </head>
 
 <body>
-  <h1>Vampire the Masquerade 5 Character Sheet Editor</h1>
```

Comment:
nemal by tam nadpis zostat?


mozno miesto Editor by bolo lepsie Generator

Reply:
podla mna to staci ze je v title

Reply:
takze tag h1 je uplne zbytocny :-)
h1 treba mat aby bolo vidiet jasne kde si, ked tam prvy raz prides, alebo pre SEO, ked tam pride google

Reply:
sa daju napisat este nejake ine riadky pre SEO, ale neviem ich, SEO som moc nestudoval
ja to berem tak, ze ked mas postavu, tak ta nejak nezajima, ze je to v nejakom generatori, alebo ze je tam nejaky text okolo, ale ze tam ide o udaje tej postavy, a tak by si podla mna nemusel vidit nic ine. no a v zaveru by sa editacna cast mala co najviac podobat read-only casti

Reply:
Nadpis je dolezita cast dokumentu. Urcuje o com ten dokument je. Treba mat velky h1 nadpis jak z pohladu SEO tak z pohladu user experience. Ty vies o com to je, ale ten kto tam pride po prvy raz nevie

Reply:
tak teda nadpis zmazeme aby sme ho pridali neskor v nadradenej komponente, ktora este neexistuje